services.py

```diff
@@ -19,3 +20,9 @@ def mark_task_as_complete(self, task_id):
         task = next((task for task in self.tasks if task.task_id == task_id), None)
         if task:
             task.mark_complete()
+
+    def get_formatted_task_details(self, task_id):
+        task = next((task for task in self.tasks if task.task_id == task_id), None)
+        if task:
```

Comment:
This new method is helpful for retrieving formatted task details. Consider adding error handling for invalid task IDs.